react-ui/src/components/Layouts/InputWorkerTimes/TimeSlot.js

```diff
@@ -0,0 +1,84 @@
+import React  from 'react';
+import { render } from 'react-dom';
+import './TimeSlot.css'; 
+import {moment} from 'moment';
+
+
+/* import {Container, Row, Col} from 'react-grid-system'; */
+
+class TimeSlot extends React.Component { // this is a class component for the checkbox feature of selecting recurring days
+  constructor() {
+    super();
+    this.state = {
+      Times: [
+        {id: 1, value: "9:00", endtime:"9:30"},
+        {id: 2, value: "9:30",endtime:"10:00"},
+        {id: 3, value: "10:00",endtime:"10:30"},
+        {id: 4, value: "10:30",endtime:"11:00"},
+        {id: 5, value: "11:00",endtime: "11:30"},
+        {id: 6, value: "11:30",endtime: "12:00"},
+        {id: 7, value: "12:00",endtime: "12:30"},
+        {id: 8, value: "12:30",endtime: "12:30"},
+        {id: 9, value: "13:00",endtime: "13:30"},
+        {id: 10, value: "13:30",endtime: "14:00"},
+        {id: 11, value: "14:00",endtime: "14:30"},
+        {id: 12, value: "14:30",endtime: "15:00"},
+        {id: 13, value: "15:00",endtime: "15:30"},
+        {id: 14, value: "15:30",endtime: "16:00"},
+        {id: 15, value: "16:00",endtime: "16:30"},
+        {id: 16, value: "16:30",endtime: "17:00"},
+        {id: 17, value: "17:00",endtime: "17:30"},
+      ],
+      checkedTimes: new Map(),
+
+    }
+  
+    this.handleChange = this.handleChange.bind(this);
+    this.handleSubmit = this.handleSubmit.bind(this);
+  }
+     
+  handleChange(event) {
+        var isChecked = event.target.checked;
+        var item = event.target.value;
+         
+        this.setState(prevState => ({ checkedTimes: prevState.checkedTimes.set(item, isChecked) }));
+  }
+     
+  handleSubmit(event) {
+    console.log("this button works"); // edit this for future functions
+    event.preventDefault();
+  }
+
```

Comment:
Hi Kyle,
Same comment as above, just make sure that unneeded whitespace is removed from the files so that they do not become unnecessarily long in length.

Reply:
> Hi Kyle,
> 
> Thank you for making the desired improvements. There are a few more tweaks that I would recommend. A helpful tip would be to use the Files Changed tab on your GitHub PR to see all the files and code that you have changed. If you noticed anything out of the ordinary or excessive whitespace, please address them in your PR (it may be side effects of resolving any merge conflicts). Additionally, please write brief unit tests for this component before we merge it into master. Great work so far!

Hi Amy, how would I exactly write a unit test?

Reply:
Hi Kyle,
I would suggest looking at the unit tests for the AppointmentList component located in the respective folder under src. Unit test files end in `.test.js`.

Reply:
Hi Amy, 
I have written some unit tests as requested.